React/my-app/src/service/accounts.js

```diff
@@ -4,10 +4,24 @@ export const postAccount = (url, body) => {
   return api.post(url, body);
 };
 
+export const getAccount = url => {
+  return api.get(url);
+};
+
 export const checkUser = body => {
   return postAccount("/users/sessions", body);
 };
 
 export const newUser = body => {
   return postAccount("/users", body);
 };
+
+export const getUser = token => {
+  api.defaults.headers.common["authorization"] = token;
+  return getAccount("/users/me");
+};
+
+export const addNewItemToWishlist = body => {
+  api.defaults.headers.common["authorization"] = window.localStorage.token;
+  return postAccount(`/users/${window.localStorage.userId}/wishes`, body);
```

Comment:
Same with `postAccount`, It does the same as api.post.
You can refactor it to just receive the `userId` and body or just remove it.